app/src/androidTest/java/org/kiwix/kiwixmobile/bookmark/BookmarksActivityTest.java

```diff
@@ -25,14 +25,14 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.kiwix.kiwixmobile.core.page.bookmark.BookmarksActivity;
+import org.kiwix.kiwixmobile.core.page.bookmark.BookmarksFragment;
 
 @RunWith(AndroidJUnit4.class)
 public class BookmarksActivityTest {
 
   @Rule
-  public ActivityTestRule<BookmarksActivity> mActivityTestRule = new ActivityTestRule<>(
-    BookmarksActivity.class);
+  public ActivityTestRule<BookmarksFragment> mActivityTestRule = new ActivityTestRule<>(
```

Comment:
That should not work

Reply:
What is this test trying to achieve? Is it already tested in MainActivityTest because it navigates to bookmarks?